registry.json

```diff
@@ -3656,5 +3656,19 @@
     "telegram": "hungbv2",
     "operatorName": "Bui Viet Hung",
     "github": "sirhung1993"
+  },
+  {
+    "ticker": "VIP",
+    "poolId": "pool1r9zrp0hpy3dj6lse5vl9yc672v5w7fzrrp62pjcersqe2djm409",
```

Comment:
This is incorrect - i think your pool id is actually 194430bee1245b2d7e19a33e52635e5328ef24431874a0cb191c0195. Please correct

Reply:
The pool Id updated